go/capture/driver/airbyte/connector.go

```diff
@@ -53,6 +54,12 @@ func RunConnector(
 		"run",
 		"--rm",
 	}
+
+	if permissiveConnectorNetworking {
+		imageArgs = append(imageArgs, "--network=host")
+		log.Infof("Granting host network access to %s container", image)
```

Comment:
nit: this log seems redundant / abstracted vs the INFO logging of the connector arguments below